static/src/js/containers/ProjectPanelsContainer/ProjectPanelsContainer.js

```diff
@@ -37,8 +37,8 @@ export const mapDispatchToProps = (dispatch) => ({
     (path) => dispatch(actions.changePath(path)),
   onChangeProjectGranulePageNum:
     (data) => dispatch(actions.changeProjectGranulePageNum(data)),
-  onFetchDataQualitySummaries:
-    (conceptId) => dispatch(actions.fetchDataQualitySummaries(conceptId)),
+  onSetDataQualitySummaries:
```

Comment:
I don't think this function is being used anywhere (or the old version). If not, it should be removed

Reply:
Function has been removed and the test accompanying it as well